app/models/agents/http_status_agent.rb

```diff
@@ -0,0 +1,89 @@
+module Agents
+
+  class HttpStatusAgent < Agent
+
+    include WebRequestConcern
+    include FormConfigurable
+
+    can_dry_run!
+    can_order_created_events!
+
+    default_schedule "every_12h"
+
+    form_configurable :url
+    form_configurable :disable_redirect_follow, type: :array, values: ['true', 'false']
+
+    description <<-MD
+    The Http Status Agent will check a url and report back the http status code.
+
+    Specify a `Url` and the Http Status Agent will produce an event with the http status code.
+
+    `Disable redirect follow` will tell the agent not follow any redirects. For example, setting this to `true` will cause an agent that receives a 301 redirect to http://yahoo.com to return a status of 301. Setting this to `false` will cause the agent to follow that 301 redirect and report the status of http://yahoo.com.
+MD
```

Comment:
Like

```ruby
  description <<-MD
    I am a description
  MD

  event_description do
    <<MD.strip_heredoc
      I am an event description
    MD
  end
```

Basically just that the ends of those heredocs can be indented, I think.  Not important.